src/main/java/org/jitsi/jicofo/SSRCValidator.java

```diff
@@ -94,6 +211,59 @@ public SSRCValidator(String               endpointId,
     }
 
     /**
+     * Because {@link SourcePacketExtension} inside of
+     * {@link SourceGroupPacketExtension} do not contain any parameters (CNAME,
+     * MSID, etc.) those have to be copied from the
+     * {@link SourcePacketExtension}s signalled in the media section in order
+     * to simplify the stream validation process by not having to look up
+     * another collection.
+     *
+     * @throws InvalidSSRCsException if a corresponding
+     * {@link SourcePacketExtension} signalled in {@link #sourceGroups} does not
+     * exist in {@link #sources}.
+     */
+    private void copySourceParamsToGroups()
```

Comment:
i think it's better to have this be a standalone function which takes in only the data it needs to read/modify, that way it keeps it more isolated. (i.e. any members it's currently accessing, make it a non-member method and just pass those fields in as arguments and modify them in place ore return the modified version which the caller can assign).

Reply:
Hmm I don't know. It modifies it's member fields and that's the intention, to modify this specific fields knowing what they hold specifically.

Reply:
I will extract this to some static utility method.